server/controllers/api/assessmentDayDataController/index.js

```diff
@@ -67,7 +67,7 @@ const AssessmentDayDataController = {
           ...sortedDayData.map((dayData) => dayData.day)
         )
         console.log('Participant data update')
-        await AssessmentDayDataModel.update(appDb, query, {
```

Comment:
should we remove the `update` method on the `AssessmentDayDataModel` if we can't use it?

Reply:
[46cd133](https://github.com/AMP-SCZ/dpdash/pull/753/commits/46cd133f33881e5a8089c4d30a2c7478471cc297)